server/handlers_v2.go

```diff
@@ -466,16 +466,17 @@ func (server HTTPServer) getImpactedClusters(
 		}
 	} else {
 		// generate JSON payload of the format "clusters": []clusters
-		jsonBody, err := json.Marshal(
+		jsonBody, e := json.Marshal(
```

Comment:
why not `err`?

Reply:
because the linter that detects ineffectual assignments complains when I reuse err everywhere, and I don't want to fight with it =)

Reply:
it looks better post-refactor